lib/layout/LayoutUtil.js

```diff
@@ -59,6 +59,29 @@ export function roundPoint(point) {
  * @return {RectTRBL}
  */
 export function asTRBL(bounds) {
+  if (isConnection(bounds)) {
```

Comment:
As the parameter is documented as `@param {Point|Rect} bounds` I wouldn't expect to get a connection here. We should look into why a connection ends up getting passed to this function.